test/e2e/global-setup.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Why add your own global-setup? The one provided by Gutenberg does handle all the defaults for you

Reply:
Is also removed. Works good, even without it.